doc/schema/attribute-types/text.md

```diff
@@ -13,43 +13,69 @@ export const Todo: PartialSchema = {
 
 ## Parameters
 
-- `default` [{String}] - The default value of the attribute.  Example: `text({default: "hello"})`
-- `required` [{Boolean=false}] - If the attribute must be provided.
+|      key      | description                                                                 | type            | optional  | default |
+| ------------- | --------------------------------------------------------------------------- | :-------------: | :-------: | :-----: |
+| `default`     | The default value of the attribute. <br/> Example: `text({default: true})`  | `Boolean`       | Yes       |         |
+| `required`    | If the attribute must be provided.  <br/> Example: `text({required: true})` | `Boolean`       | Yes       | `false` |
```

Comment:
the base data type used by text also has primary & unique - not sure if we should document them here or point to a table that has all the shared parameters

Reply:
I feel like they should be here if the other two are. I'll add them